getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -1,93 +1,81 @@
-
-4. Finally, run the ‘BlazorApp1.sln’ file by clicking the run button in the toolbar. 
-
-**For MacOS and Linux**:
-1. Open the terminal from your project directory and run the ‘dotnet watch’ command.
-
-2. Open a web browser and navigate to the appropriate address to view and interact with your Blazor application.  
-
 ### Setup using NuGet Package
 Follow these steps to install Blazor using the NuGet Package.
 
-1. Open the project inside the Visual Studio app.
+1. Open the project in Visual Studio.
+
+2. Go to `Manage Nuget Packages`, search for `BlazorFusionCharts`, and click the `Add Package` button to install the package.   
 
-2. On the Settings icon, select “Options”.
+### Installation and Including Dependencies
+To create a new Blazor application:
+1. Remove the FusionCharts package from the following locations:
+- For Windows: `C:\Users\XXX.nuget\packages`
+- For macOS and Linux: `~/.nuget/packages`
 
-3. Find the NuGet Package Manager and select Package Sources from the left panel of the Options window.
+2. Install the latest package from [Nuget.org](https://www.nuget.org/).
 
-4. click the `+` symbol in the top-right corner to add a package.
-- For the Name field, add the name of the package source as `myget.org`.
-- For the Source field, paste the myGet Package link.
-- Click “Update”, and then click “OK”.
+3. Upon successful installation, manually add the `blazor-fusionCharts.js` file from the “C:\Users\XXX.nuget\packages\blazorfusioncharts\1.0.0\staticwebassets” address to the wwwroot folder. Or from the address "~/.nuget/packages/blazorfusioncharts/1.0.0/staticwebassets/" to your app's wwwroot folder.
 
-5. Under Tools, click the “NuGet Package Manager” option and select Manage NuGet Packages for Solution.   
+4. In the `_Host.cshtml` file, add the following script tags:
+```javascript
+<script type=”text/javascript” src=”https://cdn.fusioncharts.com/fusioncharts/latest/fusioncharts.js”></script>
+<script src=”~/blazor-fusionCharts.js”></script>
+```
+In the `_Imports.razor` file, add the FusionChartsInterop as follows:
+```javascript
+<script type=”text/javascript” src=”https://cdn.fusioncharts.com/fusioncharts/latest/fusioncharts.js”></script>
+<script src=”~/blazor-fusionCharts.js”></script>
+```
+In the `Program.cs` file, add the FusionCharts service as follows:
+```javascript
+builder.Services.AddScoped<FusionCharts.FusionChartsInterop.FusionChartsService>();
+builder.Services.AddHttpClient();
+```
 
-6. Change the project source to “myget.org” in the top-right corner. 
+### Steps to run examples
+Start by referring to the Blazor documentation link for a detailed guide on setting up and installing the necessary dependencies. 
 
-7. Finally, move the browser tab of the same window, enter the credentials, and install the package "BlazorApp1".
+Clone the code repository to your local machine from [blazor-fusionCharts](https://github.com/fusioncharts/blazor-fusioncharts) and follow these simple steps to install Blazor.
```

Comment:
blazor-fusioncharts in this case, as it is different than the previous instance. In this case, it the repo name, previously it was name of a certain file.